thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomalydetection/performanceEvaluation/PerformanceEvaluateHelper.java

```diff
@@ -0,0 +1,25 @@
+package com.linkedin.thirdeye.anomalydetection.performanceEvaluation;
+
+import com.linkedin.thirdeye.datalayer.bao.MergedAnomalyResultManager;
+import com.linkedin.thirdeye.datalayer.dto.MergedAnomalyResultDTO;
+import java.util.List;
+import org.joda.time.Interval;
+
+
+public class PerformanceEvaluateHelper {
+  public static PerformanceEvaluate getPerformanceEvaluator(PerformanceEvaluationMethod performanceEvaluationMethod,
```

Comment:
Can you write a description here, what you try to achieve here. For example why you name two Id: functionId and clone Function Id, without context.